config/config.yaml

```diff
@@ -1,3 +1,3 @@
-apiVersion: imageregistry.operator.openshift.io/v1alpha1
+apiVersion: imageregistry.operator.openshift.io/v1
```

Comment:
why wasn't this part of your other PR?  (and how did your other PR pass tests w/o this, or did it?)

Reply:
nm, i see this is from the other commit you pulled in.